modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -230,14 +230,18 @@ For more information about creating an account with only the required privileges
 [id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
 == Using {product-title} with vMotion
 
-[IMPORTANT]
-====
-{product-title} generally supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.
-====
-
-If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. These references prevent affected pods from starting up and can result in data loss.
-
-Similarly, {product-title} does not support selective migration of VMDKs across datastores, using datastore clusters for VM provisioning or for dynamic or static provisioning of PVs, or using a datastore that is part of a datastore cluster for dynamic or static provisioning of PVs.
+If you intend on using vMotion in your vSphere environment, consider the following before installing a {product-title} cluster.
+
+* {product-title} supports vMotion for compute machines only.
```

Comment:
Do we need to be even more explicit and call out - "Openshift support compute-only vMotion but only for nodes designated as compute nodes in Openshift. vMotion of control-plane nodes is not supported" ?

Reply:
Every CI environment has DRS enabled. Control plane machines are vmotioned. This should be supported with the caveat of correctly setting up vmotion networks and resource capacity.

Reply:
Yep, So what does "supports vMotion for compute machines only." mean?  Does it mean, we only support compute vmotion or vmotion of only compute nodes? Also @mjpytlak we generally don't use "machine" word to refer to a individual VM/node in OCP.

Reply:
I missed that part, yeah the title `{product-title} supports vMotion for compute machines only.`
contradicts the below text. `To help ensure the uptime of your compute and control plane machines, it is recommended that you follow the VMware best practices for vMotion.`

Reply:
That sentence should be kept same as before and not changed which was:

> {product-title} generally supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.